controller/consul.go

```diff
@@ -52,7 +53,11 @@ func NewConsulMon(addr string) (*ConsulMon, error) {
 
 func (c *ConsulMon) queryServices() ([]*App, error) {
 	var apps []*App
-	addr := c.addr + fmt.Sprintf("%s/%s", nodeURL, c.node)
+	params := url.Values{}
+	if os.Getenv(allowStale) == "true" {
+		params.Add("stale")
```

Comment:
This requires two parameters, a key and a value. It might be best to just append the value "stale" like traditional string?

Reply:
You are correct, this does not work with params.Add(), reverting.